django/contrib/admin/helpers.py

```diff
@@ -116,10 +117,14 @@ def __init__(
 
     @property
     def media(self):
-        if "collapse" in self.classes:
-            return forms.Media(js=["admin/js/collapse.js"])
         return forms.Media()
 
+    @cached_property
+    def is_collapsible(self):
+        if any([error in self.fields for error in self.form.errors]):
+            return False
+        return "collapse" in self.classes
```

Comment:
Is it worth making `"collapse"` a constant somewhere? 🤔

Also, would it be better to switch to `"collapsible"`? I know this is probably kept for backward compatibility, but IIUC we'll be requiring everyone to switch to using `<details>`, so it is a breaking change anyway?

Reply:
I think this change is only really "breaking" if the user is overriding some JS or template around this. For most users nothing needs to change, so IMO not worth making everyone using this update their code.